Homework1/task4.py

```diff
@@ -0,0 +1,15 @@
+""" Задача 8: Требуется определить, можно ли от шоколадки размером n × m долек отломить k долек, 
+если разрешается сделать один разлом по прямой между дольками (то есть разломить шоколадку на два прямоугольника).
+
+*Пример:*
+
+3 2 4 -> yes
+3 2 1 -> no
+"""
+n = 3
+m = 2
+k = int(input('Какое количество долек отломить? '))
+if (k < n * m) and (k % 2 == 0) or (k % 3 == 0):
```

Comment:
В совокупности ваши условия неверны, тут нужны скобки,  но не так как вы поставили.
```suggestion
if k < n * m and (k % 2 == 0 or k % 3 == 0):
```